### ISSUE TITLE
[BUG][DART] Update Intl package version

### ISSUE THREAD

**Issue by Nikarous** [OpenAPITools/openapi-generator#8971]
#### Bug Report Checklist

- [ x] Have you provided a full/minimal spec to reproduce the issue?
- [ x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x ] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x ] Have you searched for related issues/PRs?
- [x ] What's the actual output vs expected output?

##### Description

When I use OpenApi Generator (5.0.1) to generate models for Flutter app it fails during app building with error:
```
Because every version of flutter_localizations from sdk depends on intl 0.17.0 and every version of core_design from path depends on intl ^0.16.1, flutter_localizations from sdk is incompatible with core_design from path.

So, because login depends on both core_design from path and flutter_localizations any from sdk, version solving failed.
```
This started to happen after upgrade to Flutter 2.0
##### openapi-generator version
I tested 5.0.1 and 5.0.0-beta2

##### Generation Details
java -jar openapi-generator-cli.jar generate -i ${OPENAPI_DRIVER_SPEC_FILE} -g dart -c open-generator-config.yaml --enable-post-process-file --global-property models,supportingFiles

config:
`# Config options for the dart-hhtp generator
pubName: model_generator
pubVersion: 0.0.1
pubDescription: <project name>
 `

##### Steps to reproduce

Generate models, run flutter pub get.

##### Suggest a fix
Upgrade intl package version to 0.17.0+

**Comment by wing328** [comment]
@Nikarous I wonder if you've time to file a PR to update the package to newer version.

**Comment by Nikarous** [comment]
Hi, I have the changes ready I am just waiting for another PR in different package for which OpenApi generator have dependency ;)

**Comment by Nikarous** [comment]
This will be fixed with #8869

**Comment by kuhnroyal** [comment]
@Nikarous  FYI - #8869 currently only is a PR for the `dart-dio` generator not for the `dart` generator that you are using. Like I suggested in your PR, try to override the dependency versions manually and see how far you get.

**Comment by kuhnroyal** [comment]
And can you fix the issue title please? :)

**Comment by kuhnroyal** [comment]
I was more thinking about Java -> Dart :)

**Comment by kuhnroyal** [comment]
This got addressed by #9392. The issue can be closed.
